modules/indexing/src/test/java/org/apache/ignite/internal/processors/query/LongRunningQueryTest.java

```diff
@@ -54,9 +56,33 @@ public class LongRunningQueryTest extends AbstractIndexingCommonTest {
     /** Keys count. */
     private static final int KEY_CNT = 1000;
 
+    /** Long query warning timeout. */
+    private static final int LONG_QUERY_WARNING_TIMEOUT = 1000;
+
     /** External wait time. */
     private static final int EXT_WAIT_TIME = 2000;
 
+    /** Insert. */
+    private static final String INSERT = "insert into test (_key, _val) values (1001, sleep_func_dml())";
+
+    /** Insert with a subquery. */
+    private static final String INSERT_WITH_SUBQUERY = "insert into test (_key, _val) select p._key, p.orgId from " +
+        "\"pers\".Person p where p._key < sleep_func(?)";
+
+    /** Update. */
+    private static final String UPDATE = "update test set _val = sleep_func_dml() where _key = 1";
```

Comment:
Why we can't always use sleep_func(?) and simplify code? (remove sleep_func_dml, remove isSubquery parameter)

Reply:
Done.